### PR TITLE
Launch configuration wizard mismatched label/text

### LINKED ISSUE #985: Launch config wizard checkbox header vs description mismatch
image
The description seems correct, but the header is wrong.

### LINKED ISSUE #986: Infoset diff view does not work
Not Sure where this feature came from or when it stopped working.
Inforset file that can be displayed is a temp file....
image
Solution might be to just remove this option.

### PR DESCRIPTION
**Pull request by lrbarber**
fixing text description of checkbox to open infoset viewer and infoset diff viewer and
correcting url parse/file problem that prevented diff viewer from finding files on Windows.
Closes #985 
Closes #986

### CODE REVIEW

**Comment by hdalsania** [comment]
This functionality works as expected in Linux platform. Infoset diff viewer functionality configuration is not necessary to remove from the Launch wizard. Recommend bringing infoset view diff setting back in launch configuration but fix the description for infoset viewer and infoeset diff viewer,

This functionality does not work in windows platform and that need to be fixed. It seems like a windows pathing issue as shown below

image

File: src/launchWizard/launchWizard.js [hunk 1]
```diff
@@ -60,9 +60,6 @@ function getConfigValues() {
   const tdmlDescription = document.getElementById('tdmlDescription').value
   const tdmlPath = document.getElementById('tdmlPath').value
   const openDataEditor = document.getElementById('openDataEditor').checked
-  const openInfosetDiffView = document.getElementById(
```
Comment (shanedell) [inline src/launchWizard/launchWizard.js:63]: Don't delete the infoset diff view stuff. This stuff will still be needed just updated to fix the label/text

Comment (hdalsania) [reply src/launchWizard/launchWizard.js:63]: > Don't delete the infoset diff view stuff. This stuff will still be needed just updated to fix the label/text

Agree, We should not delete that functionality as it works as intended. @lrbarber will bring that functionality back and only fix the description mismatch along with fixing the infoset.ts for Uri encoding.

**Comment by lrbarber** [comment]
Pull request has been updated.

**Comment by hdalsania** [comment]
+1 . Everything is working as expected in Windows and Linux.  @shanedell Can you verify this solution in MacOS?

**Review by hdalsania — APPROVED** [review]
+1

**Review by shanedell — APPROVED** [review]
+1 Just fix the formatting issue and squash and this PR should be good with another green +1

**Pull request merged**